docs/mine-hnt/hotspot-makers/burn-hnt-to-maker-wallet.mdx

```diff
@@ -0,0 +1,79 @@
+---
+id: burn-hnt-to-maker-wallet
+hide_title: true
+sidebar_label: Burning HNT to Maker Wallet
+slug: /mine-hnt/hotspot-makers/burn-hnt-to-maker-wallet
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+# Burning HNT to Your Maker Wallet
+
+Any [approved Hotspot Maker](/mine-hnt/approved-makers) is responsible for supplying Data Credits (DCs) to the onboarding server for their Hotspots. They do this by burning HNT to their 
+specific maker address. 
+
+:::info 
+
+A full list of approved maker wallet addresses [can be seen via this API](https://onboarding.dewi.org/api/v2/makers).
```

Comment:
Can we automatically generate a table with this API response?

Reply:
In browser? Or in the docs?